crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -18,85 +18,88 @@ pub(super) fn place_comment<'a>(
     comment: DecoratedComment<'a>,
     locator: &Locator,
 ) -> CommentPlacement<'a> {
-    // Handle comments before and after bodies such as the different branches of an if statement.
```

Comment:
For clarity: this is now down with one `or_else` chain, with the new `handle_parenthesized_comment` coming first.